documentation/jetty-documentation/pom.xml

```diff
@@ -251,6 +251,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
         <configuration>
+          <skipAssembly>${asciidoctor.skip}</skipAssembly>
```

Comment:
This deserves a comment, otherwise using a reference to an asciidoctor property in the assembly plugin config is confusing.

Reply:
done with f157e5c924544198e43187400ab668e0e899453b